Integrations/McAfee_DXL/McAfee_DXL_description.md

```diff
@@ -1,7 +1,16 @@
-  ### Prerequisites:
-   1.  [Create certificates & configure dxl](https://opendxl.github.io/opendxl-client-python/pydoc/index.html). 
-   2.  You must have the following files:
-       * Broker CA certificates (`brokercerts.crt` file)
-       * Client certificate (`client.crt` file)
-       * Client private key (`client.key` file)
-       * Broker list properties file (`brokerlist.properties` file)
+## how to configure DXL based integration
+- make sure that **openssl** is installed
+- open a new directory.
+- download the sh script https://github.com/demisto/content/raw/dxl_exemple/Integrations/McAfee_DXL/create_keys.sh.zip and move to the directory.
```

Comment:
```suggestion
- download the sh script https://github.com/demisto/content/raw/master/Integrations/McAfee_DXL/create_keys.sh.zip and move to the directory.
```

Reply:
the script is now in use will be renamed before the merge.